lxd/device/tpm.go

```diff
@@ -217,13 +217,20 @@ func (d *tpm) startVM() (*deviceConfig.RunConfig, error) {
 		},
 	}
 
-	proc, err := subprocess.NewProcess("swtpm", []string{"socket", "--tpm2", "--tpmstate", fmt.Sprintf("dir=%s", tpmDevPath), "--ctrl", fmt.Sprintf("type=unixio,path=%s", socketPath)}, "", "")
+	var fdFiles []*os.File
```

Comment:
You're not closing the files in here once the swtpm process has been started (which the comment in util.ShortenedSocketPath says to do).